CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## TBD
+
+Update plugin to search for shared object files in new artefact locations
```

Comment:
This should describe the conditions where this fixes a potential issue, like the version of bugsnag-android this affects (e.g. bugsnag-android > 4.15.0)

Reply:
Updated the changelog entry to be a bit more specific.